pytest_fixtures/api_fixtures.py

```diff
@@ -164,9 +161,8 @@ def default_os(
         )
     os.architecture.append(default_architecture)
     os.ptable.append(default_partitiontable)
-    os.config_template.append(module_configtemaplate)
-    os.provisioning_template.append(module_provisioingtemplate)
-    os.update(['architecture', 'ptable', 'config_template', 'provisioning_template'])
+    os.provisioning_template.append(module_configtemplate)
```

Comment:
`Config Template` shouldn't be available any more right ?

Reply:
It's `default_pxetemplate` :) .. it would good in latest push .